backend/src/database/database.providers.ts

```diff
@@ -0,0 +1,20 @@
+import { createConnection } from 'typeorm';
+import {DB_CONNECTION_TOKEN} from '../user/constants';
+
+export const databaseProviders = [
+  {
+    provide: DB_CONNECTION_TOKEN,
+    useFactory: async () => await createConnection({
```

Comment:
When config will be injectable, we can use it here to allow work on local db configured in `properties.json`